signac/__init__.py

```diff
@@ -30,15 +30,16 @@
 from .core.jsondict import flush_all as flush
 from .db import get_database
 from .diff import diff_jobs
-from .synced_collections.backends.collection_json import BufferedJSONDict, JSONDict
+from .synced_collections.backends.collection_json import BufferedJSONAttrDict
+from .synced_collections.backends.collection_json import JSONAttrDict as JSONDict
```

Comment:
Don't we need the buffered class here, for backwards compatibility? In my understanding, the signac 1.6.0 implementation will also buffer arbitrary `JSONDict` instances created by the user, but this would not. We would need to `import BufferedJSONAttrDict as JSONDict` if that's correct.

Reply:
... yes. Annoyingly :P